src/librustdoc/html/static/themes/dark.css

```diff
@@ -197,7 +197,7 @@ a.test-arrow {
 
 #help dt {
 	border-color: #bfbfbf;
-	background: #fff;
+	background: rgba(0,0,0,0);
```

Comment:
This creates black-on-black text for the keyboard shortcut keys on the help window.

Reply:
Ohh, i didn't realize that the help window keys got switched over to `<kbd>` tags. This is fine.